### PR TITLE
Proselytizers can now charge from Sigils of Transmission, and Sigils of Transmission can be charged with brass

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Proselytizers can now charge from Sigils of Transmission at a rate of 1000W per second.
rscadd: Sigils of Transmission can be charged with brass at a rate of 250W per sheet.
/:cl:

Vitality Matrices now handle their animates slightly better via ANIMATION_END_NOW

### CODE REVIEW

**Comment by Armhulen** [comment]
holy shit yes

File: code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm [hunk 1]
```diff
@@ -202,6 +202,21 @@
 		and <b>[structure_number]</b> Clockwork Structure[structure_number == 1 ? "":"s"] [structure_number == 1 ? "is":"are"] in range.</span>")
 		if(iscyborg(user))
 			to_chat(user, "<span class='brass'>You can recharge from the [sigil_name] by crossing it.</span>")
+		else if(GLOB.ratvar_awakens)
```
Comment (lzimann) [inline code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm]: This double `if(GLOB.ravtar_awakens)` bothers me a lot.

File: code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm [hunk 2]
```diff
@@ -202,6 +202,21 @@
...
+
+/obj/effect/clockwork/sigil/transmission/attackby(obj/item/I, mob/living/user, params)
+	if(is_servant_of_ratvar(user) && istype(I, /obj/item/stack/tile/brass) && !GLOB.ratvar_awakens)
+		var/obj/item/stack/tile/brass/B = I
+		user.visible_message("<span class='warning'>[user] places [B] on [src], causing it to disintegrate into glowing orange energy!</span>", \
+		"<span class='brass'>You charge the [sigil_name] with [B], providing it with <b>[B.amount * POWER_FLOOR]W</b> of power.</span>")
+		modify_charge(-(B.amount * POWER_FLOOR))
+		playsound(src, 'sound/effects/light_flicker.ogg', (B.amount * POWER_FLOOR) * 0.01, 1)
+		qdel(B)
+		return 1
```
Comment (lzimann) [inline code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm]: TRUE

File: code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm]: if you can figure out a better way to do it let me know

**Pull request merged**